wondrous-app/components/Collaboration/SharedOrgHeader/index.tsx

```diff
@@ -4,7 +4,85 @@ import { SafeImage } from 'components/Common/Image';
 import { DAOEmptyIcon } from 'components/Icons/dao';
 import { TokenEmptyLogo } from 'components/organization/wrapper/styles';
 import Link from 'next/link';
-import { Separator } from './styles';
+import { OrgContainer, OrgName, Separator, SeparatorDiv, SmallSeparator } from './styles';
+
+export const SharedOrgHeaderCard = (props) => {
+  const { collab, blurChild, blurParent } = props;
+  return (
+    <Grid
+      container
+      direction="row"
+      wrap="nowrap"
+      gap="8px"
+      sx={{ width: 'auto' }}
+      style={{
```

Comment:
We can pass width and alignItems directly to Grid as props, something like `<Grid alignItems="center" width="auti" .../>`

Reply:
https://mui.com/material-ui/api/grid/ doesn't seem like it?